Add Unagi_Gar-Lampli

dusjagr opened the pull request.

kasbah commented:
> hab am weekend nochmals den diy-CAD workshop gemacht und super outputs. das
> eine grad aufgearbeitet zur fertigung. mit allem drum und dran...

>scheck mal wann es bei dir online ist:
>https://github.com/GenericLab/Unagi_Gar-Lampli

Doller Fisch! preview: http://add-unagi.preview.kitnic.it/boards/github.com/GenericLab/Unagi_Gar-Lampli/

Willste damit den anderen GaerLaempli ersetzen oder den da lassen und diesen dazufuegen?

>habs mal hier visualisiert: http://viewer.tracespace.io/
>ist meega, kennst du was gutes für offline?
>kämpfe üüübel mit kicad.

Ich benutzt normal Gerbv fuer offline aber man koennte auch die viewer.tracespace.io Seite einfach speichern weil das ganz im Browser laueft (also auch offline).

kasbah commented:
P.S. What's the difference between SK6812 and AA3528SURSK and should there be both?

dusjagr commented:
Cool,
Lassen wir das andere design mal da aus historischen gründen...

M



On 5 Apr 2017 22:52, "Kaspar Emanuel" <notifications@github.com> wrote:

hab am weekend nochmals den diy-CAD workshop gemacht und super outputs. das
eine grad aufgearbeitet zur fertigung. mit allem drum und dran...

scheck mal wann es bei dir online ist:
https://github.com/GenericLab/Unagi_Gar-Lampli

Doller Fisch! preview: http://add-unagi.preview.kitnic.it/boards/github.com/
GenericLab/Unagi_Gar-Lampli/

Willste damit den anderen GaerLaempli ersetzen oder den da lassen und
diesen dazufuegen?

habs mal hier visualisiert: http://viewer.tracespace.io/
ist meega, kennst du was gutes für offline?
kämpfe üüübel mit kicad.

Ich benutzt normal Gerbv fuer offline aber man koennte auch die
viewer.tracespace.io Seite einfach speichern weil das ganz im Browser
laueft (also auch offline).

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/monostable/kitnic/pull/97#issuecomment-291886636>, or mute
the thread
<https://github.com/notifications/unsubscribe-auth/AAV0poNoI5mmHsHSLBkR2ywKYV8t1_M1ks5rs6qUgaJpZM4MzxYp>
.

kasbah commented:
>P.S. What's the difference between SK6812 and AA3528SURSK and should there be both?

dusjagr commented:
yip, it's correct.
1 is the normal status LED, top led, single color.

the others are the neo-pixels, but i don't have any supplier. you really
should add that taobao button :-)

m

On Wed, Apr 5, 2017 at 11:02 PM, Kaspar Emanuel <notifications@github.com>
wrote:

> P.S. What's the difference between SK6812 and AA3528SURSK and should there
> be both?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/monostable/kitnic/pull/97#issuecomment-291890027>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAV0pn6ODN5-Mi_n-fzs_jPusV3DUxDNks5rs6zwgaJpZM4MzxYp>
> .
>



-- 
*******************************************************
!!NOTICE: new phone nr:  tel: +41 77 9930877
*******************************************************
//////dusjagr labs/////////////////////////////////////////
Dr. Marc Dusseiller
Schöneggstr. 34 | CH-8004 Zürich | tel: +41 77 9930877 | skype: dusjagr
www.dusseiller.ch/labs
//////////////////////////////////////////////////////////////////
SGMK | Schweizerische Gesellschaft für Mechatronische Kunst
Postfach 2161 | CH-8031 Zürich
www.mechatronicart.ch
//////////////////////////////////////////////////////////////////
hackteria | Open Source Biological Art
www.hackteria.org
//////////////////////////////////////////////////////////////////

dusjagr commented:
fixed some other things in the BOM.

switched to DIL for the attiny and added a socket.
and corrected the audio jack (just ordered 700 from tao bao, 5cents a
piece, and in differenc colors)

m

On Thu, Apr 6, 2017 at 9:45 AM, Marc Dusseiller <marc@dusseiller.ch> wrote:

> yip, it's correct.
> 1 is the normal status LED, top led, single color.
>
> the others are the neo-pixels, but i don't have any supplier. you really
> should add that taobao button :-)
>
> m
>
> On Wed, Apr 5, 2017 at 11:02 PM, Kaspar Emanuel <notifications@github.com>
> wrote:
>
>> P.S. What's the difference between SK6812 and AA3528SURSK and should
>> there be both?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/monostable/kitnic/pull/97#issuecomment-291890027>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AAV0pn6ODN5-Mi_n-fzs_jPusV3DUxDNks5rs6zwgaJpZM4MzxYp>
>> .
>>
>
>
>
> --
> *******************************************************
> !!NOTICE: new phone nr:  tel: +41 77 9930877 <+41%2077%20993%2008%2077>
> *******************************************************
> //////dusjagr labs/////////////////////////////////////////
> Dr. Marc Dusseiller
> Schöneggstr. 34 | CH-8004 Zürich | tel: +41 77 9930877
> <+41%2077%20993%2008%2077> | skype: dusjagr
> www.dusseiller.ch/labs
> //////////////////////////////////////////////////////////////////
> SGMK | Schweizerische Gesellschaft für Mechatronische Kunst
> Postfach 2161 | CH-8031 Zürich
> www.mechatronicart.ch
> //////////////////////////////////////////////////////////////////
> hackteria | Open Source Biological Art
> www.hackteria.org
> //////////////////////////////////////////////////////////////////
>



-- 
*******************************************************
!!NOTICE: new phone nr:  tel: +41 77 9930877
*******************************************************
//////dusjagr labs/////////////////////////////////////////
Dr. Marc Dusseiller
Schöneggstr. 34 | CH-8004 Zürich | tel: +41 77 9930877 | skype: dusjagr
www.dusseiller.ch/labs
//////////////////////////////////////////////////////////////////
SGMK | Schweizerische Gesellschaft für Mechatronische Kunst
Postfach 2161 | CH-8031 Zürich
www.mechatronicart.ch
//////////////////////////////////////////////////////////////////
hackteria | Open Source Biological Art
www.hackteria.org
//////////////////////////////////////////////////////////////////
